release/chocolatey/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,35 @@
+$ErrorActionPreference = 'Stop';
```

Comment:
Have you tested _not_ having this file?  Since this is a standard installation of an exe, it is likely that Chocolatey' auto uninstaller can take care of removing it, and therefore this file may not be required at all, and can simply be removed.